### DISCUSSION TITLE
Inglês  Device Control in the PRO license

### DISCUSSION THREAD

**Discussion by lucasalberti** [rustdesk/rustdesk#8051] [Ideas]
I wanted to use the PRO license but we encountered a problem, instead of RustDesk controlling the number of simultaneous Connections in Devices, it registers anyone who opens the client as a Device, which doesn't make much sense, the control should be the same as for others tools like Teanviewer, LogMein etc

We want to use the PRO version with more than 50 users, but this part of the devices has become unusable.

**Comment by rustdesk** [comment]
You can build a outgoing only custom client, this won't register an id.

**Comment by lucasalberti** [comment]
In this case, my client can give access and when I access his computer he will not register on the device
